### PR TITLE
Fix typos2

### PR DESCRIPTION
**Pull request by nnsW3**
This PR addresses several typographical errors across various files in the project. The changes improve readability and maintain the professional standard of the documentation and code comments.

Justification
Typographical errors, while minor, can detract from the overall quality of the project. Correcting these errors ensures clarity and professionalism, making the project more accessible and understandable for current and future contributors.

### CODE REVIEW

File: docs/src/developer_guide/contributing_to_fuelup.md [hunk 1]
```diff
@@ -51,7 +51,7 @@ If you wish to make a significant change that alters current behavior or touches
 This is a rough outline of what a contributor's workflow looks like:
 
 - Make sure what you want to contribute is already tracked as an issue.
-  - We may discuss the problem and solution in the issue.
+  - We may discuss the problem and solution to the issue.
```
Comment (JoshuaBatty) [inline docs/src/developer_guide/contributing_to_fuelup.md:54]: The original made more sense. In the issue here is referring to a specific github issue where discussion will take place within.

File: tests/testcfg/mod.rs [hunk 2]
```diff
@@ -51,7 +51,7 @@ pub struct TestCfg {
     /// <TMP_DIR>/.fuelup/bin/. This should be used to execute other binaries (eg. forc) in the
     /// test environment.
     pub fuelup_bin_dirpath: PathBuf,
-    /// The path to the test environment's home. This should usually be a created
```
Comment (JoshuaBatty) [inline tests/testcfg/mod.rs:54]: the original was correct. Referring to _a_ `tempfile::tempdir::TempDir`

**Review by JoshuaBatty — CHANGES_REQUESTED** [review]
Thanks for the PR. just 2 that need rolling back. Happy to approve after that

File: tests/testcfg/mod.rs (hunk 2 above)
Comment (nnsW3) [reply tests/testcfg/mod.rs]: change it

File: docs/src/developer_guide/contributing_to_fuelup.md (hunk 1 above)
Comment (nnsW3) [reply docs/src/developer_guide/contributing_to_fuelup.md]: done sir

**Review by JoshuaBatty — APPROVED** [review]

**Pull request merged**